packages/netsuite-adapter/src/custom_references/weak_references/fields_references.ts

```diff
@@ -187,12 +209,10 @@ const fixIndexInMappedList = (mappedList: MappedList): void =>
     })
 
 const fixIndexes = (form: InstanceElement, pathsToRemove: ElemID[]): void => {
-  const pathsToFixIndex = new Set<string>(
-    pathsToRemove.map(path => form.elemID.getRelativePath(path.createParentID()).join('.')),
-  )
+  const pathsToFixIndex = new Set<ElemID>(pathsToRemove.map(path => path.createParentID()))
```

Comment:
set doesn't work well with non primitive values...
```suggestion
  const pathsToFixIndex = _.uniqBy(pathsToRemove.map(path => path.createParentID()), id => id.getFullName())
```